MOBT-3-Patterns/MVVMWithoutC/SignOut/SignOutViewModel.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  SignOutViewModel.swift
+//  MVVMWithoutC
+//
+//  Created by Mykyta Babanin on 09.07.2021.
+//
+
+import Foundation
+import Alamofire
+
+protocol ViewModelProtocol {
+    func displayLogin()
+    func viewDidDisapear()
+    func saveLoggedState()
+}
+
+class SignOutViewModel: ViewModelProtocol {
+    
+    weak var coordinator: signOutCoordinator?
+    
+    func saveLoggedState() {
+        let def = UserDefaults.standard
```

Comment:
We can reuse the user session data provider/service (that I mentioned before) here